metricbeat/module/mongodb/mongodb.go

```diff
@@ -0,0 +1,63 @@
+package mongodb
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/mb/parse"
+
+	mgo "gopkg.in/mgo.v2"
+)
+
+func init() {
+	// Register the ModuleFactory function for the "mysql" module.
+	if err := mb.Registry.AddModule("mongodb", NewModule); err != nil {
+		panic(err)
+	}
+}
+
+func NewModule(base mb.BaseModule) (mb.Module, error) {
+	// Validate that at least one host has been specified.
+	config := struct {
```

Comment:
+1 on making it a list of params instead of a single one, as I also think we will add more int he future.